framework/core/tests/integration/api/groups/ListTest.php

```diff
@@ -172,10 +172,10 @@ public function paginates_groups_with_filter()
     {
         $response = $this->send(
             $this->request('GET', '/api/groups')
-            ->withQueryParams([
-                'filter' => ['hidden' => 1],
-                'page' => ['limit' => '1', 'offset' => '1'],
-            ])
+                ->withQueryParams([
+                    'filter' => ['hidden' => 1],
+                    'page' => ['limit' => '1', 'offset' => '1'],
+                ])
```

Comment:
nitpick: just to keep the git history clean
```suggestion
            ->withQueryParams([
                'filter' => ['hidden' => 1],
                'page' => ['limit' => '1', 'offset' => '1'],
            ])
```